superset-frontend/src/features/databases/UploadDataModel/index.tsx

```diff
@@ -173,6 +173,12 @@ const allowedExtensionsToAccept = {
   columnar: '.parquet, .zip',
 };
 
+const extensionsToLabel: Record<UploadType, string> = {
+  csv: 'CSV',
```

Comment:
did a double take on whether this needs to be wrapped with a18n, but i think it's right to NOT make these labels translatable